ov/firmware_drivers.go

```diff
@@ -149,12 +149,15 @@ func (c *OVClient) GetFirmwareBaselineById(id string) (FirmwareDrivers, error) {
 
 func (c *OVClient) GetFirmwareBaselineByNameandVersion(name string) (FirmwareDrivers, error) {
 	var fwname, version string
-	fwNameVersion := strings.SplitAfter(name, ",")
-	if len(fwNameVersion) < 1 {
+	fwNameVersion := strings.Split(name, ",")
+	if len(fwNameVersion) == 0 {
```

Comment:
I think this condition appear when we have empty string...as per doc "If s does not contain sep and sep is not empty, Split returns a slice of length 1 whose only element is s", have we taken care this scenario

Reply:
Without separator the Split function will return only the string